cite.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>the title</title>
+  <style type="text/css">
+   .box {background:lightgray;width:50%}
+   .note {background:red;width:75%;font-size:large}
+  </style>
+  </head>
+  <body>
+
+<p class="note">NOTE: The most recent version of this document is always available at:<br/>
```

Comment:
Can you indent the html?

Style: we don't have css files lying around anywhere, so we probably can't use a particular format. But I wonder whether we can use some common style that's around on the web?